UKHomeOffice/cerberus-service
# COP-8933: Add tests to verify Back to task list workflow

## Description
Add tests to verify Back to task list workflow
Update delete automation data method so that it deletes all the data after weekend, check if the day is Monday and subtract 3 days from the current data to delete the tasks.
Add / update tests to check the passenger information on different versions where one of the versions doesn't contain passenger data and other 2 versions have passenger data.

## To Test
npm run cypress:runner
run test `Should verify Back to task list workflow` from spec `task-management.spec.js`.
run following tests from spec `task-version-details.spec.js`
`Should verify single task created for the same target with different versions with different passengers information`
`Should verify passenger details on different version on task details page`
 

## Developer Checklist

\* Required
